utilities/data/getStoryList.ts

```diff
@@ -38,9 +39,9 @@ async function getStoryList({ path }: getStoryDataProps) {
 
   // For more related documentation see app/(storyblok)/[[...slug]]/page.tsx
   const sbParams: ISbStoriesParams = {
-    version: 'published',
-    cv: activeEnv === 'development' ? Date.now() : undefined,
-    starts_with: 'stories/',
+    version: isProd ? 'published' : 'draft',
```

Comment:
I originally set this as always be "published", ie, only displaying list of published stories when viewing the filter pages even when in the editor, but I suppose this is ok too 🤔 😄